Watchman.Discord/Areas/Administration/Controllers/DateTimeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+namespace Watchman.Discord.Areas.Administration.Controllers
+{
+    public static class DateTimeExtensions
+    {
+        public static string ToLocalTimeString(this DateTime dateTime)
+        {
+            return TimeZoneInfo.ConvertTimeFromUtc(dateTime, TimeZoneInfo.Local).ToString("dd.MM.yyyy HH:mm:ss", CultureInfo.InvariantCulture);
```

Comment:
jak dobrze pamiętam to w dockerze jako lokalny mamy UTC
możliwe że @gagyn coś zmieniał i nie pamiętam

myśle że do obsługi stref czasowych lepszą opcją będzie na sztywno wpisywanie nazw stref czasowych
a jako extensions metody typu ToCestTime() ToWestTime() ToGmtTime() itd

Reply:
możliwe że w przyszłości będzie tak
że serwery będą mogły sobie skonfigurować strefe czasową więc to też trzeba brać pod uwage

Reply:
tak jak jest tutaj tak jest teraz i działa, więc lepiej nie zmieniać